uninstall.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// if uninstall.php is not called by WordPress, die
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
+}
+
+delete_option( 'wc_connect_remote_notices' );
```

Comment:
Nice! This is an area we're currently deficient in. See: https://github.com/Automattic/woocommerce-connect-client/issues/776